packages/features/auth/lib/next-auth-options.ts

```diff
@@ -342,7 +342,8 @@ if (isSAMLLoginEnabled) {
           return null;
         }
 
-        const { id, firstName, lastName, email } = userInfo;
+        const { id, firstName, lastName } = userInfo;
+        const email = userInfo.email.toLowerCase();
```

Comment:
We convert to lower case to ensure an uppercase email doesn't slip in from the idP